flit/install.py

```diff
@@ -266,6 +271,13 @@ def install_directly(self):
 
         self.write_dist_info(dirs['purelib'])
 
+    def add_pth_entry(self, src, dst):
+        log.info("pth %s %s", src, dst)
```

Comment:
This log call is superfluous - the same info is logged just before.

Reply:
Since this is a simple thing, I'm going to merge it and remove the log call myself. Thanks for your contribution!